ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,49 @@
+name: '🐞 Bug report'
+description: Create a report to help us improve Nuxt
+labels: ['pending triage']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please carefully read the contribution docs before creating a bug report
+        👉 https://nuxt.com/docs/community/reporting-bugs
+
+        Please use a template below to create a minimal reproduction
+        👉 https://stackblitz.com/github/nuxt/starter/tree/v3
```

Comment:
Do you think this (and other references) could be made generic? Maybe using a GitHub variable to reference the repo this is being used in?

Reply:
Is this what you have in mind?

```
Please use a template below to create a minimal reproduction
👉 https://stackblitz.com/github/${repo_name}
👉 https://codesandbox.io/s/github/${repo_name}
```

I would also think this would be ideal, but I don't have a solution at the moment.

I think it's possible to create an issue using GitHub Actions, but I think it would be hard to use variables when using templates.
It would be nice if it could be achieved with Autolinks, but I haven't tried it yet...

- https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources

Reply:
Another solution is to have the user enter stackblitz as input.
For example, Nuxt Scripts asks the user to enter it.

https://github.com/nuxt/scripts/blob/3c8838558debe268861e33e8a5abba02523aa36e/.github/ISSUE_TEMPLATE/bug-report.yml#L12-L18

Reply:
Exactly - I'd like to have the repo autofilled in the text of the template.

Reply:
I'm glad we were on the same idea.

I'll continue to investigate and consider this renovation.